src/service.ts

```diff
@@ -25,4 +26,10 @@ export default async function init({
       };
     }
   });
+
+  iris.register({
+    pattern: 'action.annotate.metis',
```

Comment:
Same as in the previous comment, let's avoid coupling to the name of the service. The action is enough.